src/index.js

```diff
@@ -104,6 +105,8 @@ class RelayCompilerWebpackPlugin {
       sourceParserName,
       languagePlugin: language
     })
+
+    this.languagePlugin = language
```

Comment:
It wasn't obvious to me how I should pass the language output extension down to compile but I figured I'd just do it like this for now.